src/Mod/Spreadsheet/Gui/SheetTableView.cpp

```diff
@@ -182,6 +240,19 @@ void SheetTableView::insertRows()
     Gui::Command::doCommand(Gui::Command::Doc, "App.ActiveDocument.recompute()");
 }
 
+void SheetTableView::insertRowsAfter()
+{
+    assert(sheet != 0);
+    const auto rows = selectionModel()->selectedRows();
+    const auto & [min, max] = selectedMinMaxRows(rows);
```

Comment:
This also causes a compiler warning because the variable min is unused. The assert() below is only visible in a debug build but a NOP in release builds.